tpl/template_funcs_test.go

```diff
@@ -1799,10 +1799,13 @@ func TestHighlight(t *testing.T) {
 func TestInflect(t *testing.T) {
 	for i, this := range []struct {
 		inflectFunc func(i interface{}) (string, error)
-		in          string
+		in          interface{}
 		expected    string
 	}{
 		{humanize, "MyCamel", "My camel"},
+		{humanize, "41", "41st"},
+		{humanize, 103, "103rd"},
```

Comment:
I'm adding the int64 test case but we cannot support int32 as easily here because the underlying `cast.ToStringE` doesn't support it either:
https://github.com/spf13/cast/blob/master/caste.go#L225-L262
https://github.com/spf13/cast/blob/master/cast_test.go#L45-L60